_static/api-examples.md

```diff
@@ -126,7 +126,7 @@ You must authenticate with the CDX CROMERR service for signing a manifest. Value
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @auth-creds.json \
-  https://e-manifest.18f.gov/api/0.1/user/authenticate
+  https://e-manifest.18f.gov/api/v1/users/authentications
```

Comment:
is plural appropriate here? A request only ever operates on a single `user`.

I realize we want to keep verbs out of the URL itself, but `authentications` feels a little awkward to me. REST semantics aren't gospel.